ext/openssl/ossl_pkey.c

```diff
@@ -457,6 +457,43 @@ ossl_pkey_check_public_key(const EVP_PKEY *pkey)
     ossl_raise(ePKeyError, "public key missing");
 }
 
+/*
```

Comment:
Please move this code block to after ossl_pkey_public_to_pem() so that the ordering match with rb_define_method()'s.